README.md

```diff
@@ -8,10 +8,16 @@ All pods automatically come with runpodctl installed with a pod-scoped API key!
 ## install linux/osx command line
 get latest binary from [releases](https://github.com/Run-Pod/runpodctl/releases)
 
+linux
 ```
 wget --quiet --show-progress https://github.com/Run-Pod/runpodctl/releases/download/v1.6.1/runpodctl-linux-amd -O runpodctl && chmod +x runpodctl && sudo cp runpodctl /usr/bin/runpodctl
```

Comment:
```suggestion
wget --quiet --show-progress https://github.com/Run-Pod/runpodctl/releases/download/v1.6.1/runpodctl-linux-amd -O runpodctl && chmod +x runpodctl && sudo mv runpodctl /usr/bin/runpodctl
```

Reply:
Should it just default to mv so there are not left over artifacts?

Reply:
sure, good idea